src/config/api.ts

```diff
@@ -0,0 +1,9 @@
+import { getEnv } from "../utils/helperFunctions/env";
+
+const API_URL = getEnv("VITE_API_BASE_URL");
+
+const API_ENDPOINTS = {
+  fileSubmissions: `${API_URL}/fileSubmissions`,
```

Comment:
This is where we would add the upload url (transferred from what was the env var).